### DISCUSSION TITLE
image and sharp plugins throw error while developing, but gatsby cloud builds fine

### DISCUSSION THREAD

**Discussion by kemalgencay** [gatsbyjs/gatsby#34684] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

image and sharp plugins throw err while developing, but cloud builds fine #34628

To continue developing I had to comment out the image and sharp plugin lines from gatsby-config.js, then before pushing to github I remove the comments so that gatsby cloud builds.

### Reproduction Link

https://github.com/kemalgencay/egitim

### Steps to Reproduce

1.
2.
3.
..One of support gentleman responding to my mail wrote he also experienced the bug.

### Expected Result

I expect development server not to throw error and abort

### Actual Result

when I start gatsby develop, during the loading plugins phase it throws Illegal instruction and aborts!

### Environment

```shell
System:
    OS: Linux 5.4 Peppermint 10 (Ten)
    CPU: (2) x64 Pentium(R) Dual-Core CPU       T4500  @ 2.30GHz
    Shell: 4.4.20 - /bin/bash
  Binaries:
    Node: 16.13.2 - ~/.config/nvm/versions/node/v16.13.2/bin/node
    npm: 8.3.2 - ~/.config/nvm/versions/node/v16.13.2/bin/npm
  Languages:
    Python: 2.7.17 - /usr/bin/python
  Browsers:
    Firefox: 96.0
  npmPackages:
    gatsby: ^4.6.0 => 4.6.0 
    gatsby-plugin-image: ^2.6.0 => 2.6.0 
    gatsby-plugin-mdx: ^3.6.0 => 3.6.0 
    gatsby-plugin-sharp: ^4.6.0 => 4.6.0 
    gatsby-source-filesystem: ^4.6.0 => 4.6.0 
  npmGlobalPackages:
    gatsby-cli: 4.6.0
```

**Comment by LekoArts** [comment]
Hi!

You'll need to be more specific than "when I start gatsby develop, during the loading plugins phase it throws Illegal instruction and aborts!". Please paste the full error output.

My best guess is that you forgot to add the necessary packages for `gatsby-plugin-image` as I don't see `gatsby-transformer-sharp` installed and used.

**Reply by kemalgencay** [reply]
Thanks for your attention
Pushed the latest commit
This is the error when reading config file:
**open and validate gatsby-configs, load plugins
Illegal instruction**
BTW before developing I removed node_modules and package-lock.js and issued npm install
Cloud builds fine, maybe its a different "machine" than what I have locally

**Comment by kemalgencay** [comment]
Hi, should I expect a solution, or forget about the issue that development server throws  error and aborts when reading the mentioned plugins?
Thanks

**Reply by kemalgencay** [reply]
So sorry for the inconvenience, finally discovered the err is caused by HW memory defect, 
Regards
